src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -21,11 +21,12 @@
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. List is sorted after successfully adding a person.\n"
```

Comment:
This line is a tad bit too long. You can set the IDE to wrap lines after ~100 characters.